.github/workflows/qa.yaml

```diff
@@ -0,0 +1,65 @@
+name: QA
+on:
+  push:
+    branches:
+      - "main"
+      - "feature/*"
+      - "hotfix/*"
+      - "release/*"
+      - "renovate/*"
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    uses: canonical/starflow/.github/workflows/lint-python.yaml@main
+  test-basic:
+    # uses: canonical/starflow/.github/workflows/test-python.yaml@main
+    uses: canonical/starflow/.github/workflows/test-python.yaml@work/custom-markers
+    with:
+      # Fast tests run on several Ubuntu platforms and macos on ARM
+      fast-test-platforms: '["ubuntu-24.04", "ubuntu-24.04-arm", "macos-13"]'
+      fast-test-python-versions: '["3.10", "3.12", "3.13"]'
+      # Slow tests (which include everything that needs multipass) run on several Ubuntu platforms
+      slow-test-platforms: '["ubuntu-24.04", "ubuntu-24.04-arm"]'
+      slow-test-python-versions: '["3.12"]'
+      lowest-python-platform: "ubuntu-22.04"
+      lowest-python-version: "3.10"
+      use-lxd: true
+      pytest-markers: not lxd_instance and not multipass_instance
+  test-lxd:
+    # uses: canonical/starflow/.github/workflows/test-python.yaml@main
+    uses: canonical/starflow/.github/workflows/test-python.yaml@work/custom-markers
+    with:
+      # All fast tests run in the basic job
+      fast-test-platforms: "[]"
+      slow-test-platforms: '["ubuntu-24.04", "ubuntu-24.04-arm"]'
+      slow-test-python-versions: '["3.12"]'
+      lowest-python-platform: ""
+      use-lxd: true
+      pytest-markers: lxd_instance
+  # test-multipass:
```

Comment:
There are still issues with multipass on GitHub's hosted runners and disabling this isn't disabling tests we were previously running anyway.